source/fundamentals/indexes.txt

```diff
@@ -313,42 +332,97 @@ the following:
 
 To learn more, see :manual:`Unique Indexes </core/index-unique>`.
 
-List Indexes
-------------
+Search Indexes
+--------------
 
-You can use the ``listIndexes()`` method to list all of the indexes
-for a collection. The `listIndexes() <{+api+}/classes/Collection.html#listIndexes>`__ method takes an
-optional `ListIndexesOptions
-<{+api+}/interfaces/ListIndexesOptions.html>`__ parameter. The ``listIndexes()`` method returns an
-object of type `ListIndexesCursor
-<{+api+}/classes/ListIndexesCursor.html>`__.
+Atlas Search is a feature that allows you to perform full-text
+searches. To learn more, see the :ref:`Atlas Search <fts-top-ref>`
+documentation.
 
-The following code uses the ``listIndexes()`` method to list all the
-indexes in a collection:
+Before you can perform a search on an Atlas collection, you must first
+create an Atlas Search index on the collection. An Atlas Search
+index is a data structure that categorizes data in a searchable format.
 
-.. literalinclude:: /code-snippets/indexes/listIndexes.js
+Starting in version 5.7 of the {+driver-short+}, you can manage your
+Search indexes using the following methods: 
+
+- ``createSearchIndex()``
+- ``createSearchIndexes()``
+- ``listSearchIndexes()``
+- ``updateSearchIndex()``
+- ``dropSearchIndex()``
+
+The following sections provide example code using each of the preceding
+methods to manage Search indexes.
+
+Create Search Indexes
```

Comment:
S: Change each section title to match the method it's demonstrating. So this one would be "Create a Search Index". Applies to all these sections.